app/portainer/rbac/components/access-viewer/access-viewer.html

```diff
@@ -1,6 +1,6 @@
 <div class="col-sm-12" style="margin-bottom: 0px">
   <rd-widget>
-    <rd-widget-header icon="fa-user-lock">
+    <rd-widget-header icon="user" feather-icon="true">
```

Comment:
I think the icon here is wrong, according to our guide it should be a custom icon called `user-lock`
![image](https://user-images.githubusercontent.com/12197793/182048877-f8e05704-2143-4131-adab-2c32f9e69866.png)